src/views/home/bestReliablePricingPlans/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Heading } from "../Heading/Heading";
+import { OptionList } from "../OptionList/OptionList";
+import { BsBoxArrowInRight } from "react-icons/bs";
+import { PrimaryLink } from "../../../../components/buttons/PrimaryLink";
+
+type Props = {
+  name: string;
+  price: string;
+  icon: React.ReactNode;
+  options: {
+    text: string;
+    isAvailable: boolean;
+  }[];
+};
+
+export const Card: React.FC<Props> = ({ name, price, icon, options }) => {
```

Comment:
No i właśnie, pytanie czy takie zagnieżdżanie elementów faktycznie ma sens, czy będziecie gdzieś indziej używać heading? Nie jest to  jakiś ogromny komponent żeby go przenosić, a pytanie czy faktycznie jest aż tak reużywalny. Aktualnie musicie dublować typy i przekazywać je niżej w propsach taki trochę nonsense